tests/boards/ehl_crb/gpio/testcase.yaml

```diff
@@ -0,0 +1,8 @@
+#Copyright(c)2021 Intel Corporation
+#
+#SPDX - License - Identifier : Apache - 2.0
+
+tests:
+  boards.ehl_crb.gpio:
+    platform_allow: ehl_crb
+    tags: gpio
```

Comment:
@udivya73 Thanks for including a sample! I'm seeing this is actually located in tests/boards/ - if your intent was to include a sample, please move it to samples/boards/ instead (adjusting dependencies as needed). Example: https://github.com/zephyrproject-rtos/zephyr/tree/master/samples/boards/up_squared/gpio_counter

Reply:
Yes, my intent was to include a sample. I will move it.